kolibri/core/assets/src/api-resources/index.js

```diff
@@ -21,6 +21,8 @@ export { default as ExamResource } from './exam';
 export { default as FacilityDatasetResource } from './facilityDataset';
 export { default as UserProgressResource } from './userProgress';
 export { default as UserSyncStatusResource } from './userSyncStatus';
+export { default as ContentDownloadRequestResource } from './contentDownloadRequest';
+export { default as ContentRemovalRequestResource } from './contentRemovalRequest';
```

Comment:
This isn't referenced anywhere, and from my understanding the removal requests are being handled in the delete handler for ContentDownloadRequests?